### PR TITLE
In normalization doc, fix incorrect info re period conversion

### PR DESCRIPTION
**Pull request by genwhittTTD**
In normalization doc, fix incorrect info re period conversion

### CODE REVIEW

**Review by sunnywu — APPROVED** [review]

File: docs/getting-started/gs-normalization-encoding.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ To normalize an email address, complete the following steps:
 
 1. Remove leading and trailing spaces.
 2. If there are uppercase characters, convert them to lowercase.
-3. If there is a period (`.`) in the email address (ASCII decimal code 46/UTF-8 hexadecimal code 2E), remove it.<br/>For example, normalize `jane.doe@example.com` to `janedoe@example.com`.
+3. (Conditional) In `gmail.com` addresses only, if there is a period (`.`) in the address (ASCII decimal code 46/UTF-8 hexadecimal code 2E), remove it.<br/>For example, normalize `jane.doe@gmail.com` to `janedoe@gmail.com`.
```
Comment (jon8787) [inline docs/getting-started/gs-normalization-encoding.md:46]: per slack comment - should we put the gmail info together?

File: docs/getting-started/gs-normalization-encoding.md [hunk 2]
```diff
@@ -114,8 +114,10 @@ Some of the examples show email addresses that include the plus sign (+), with d
...
-| `JANESAOIRSE@example.com`<br/>`JaneSaoirse@example.com`<br/>`Jane.Saoirse@example.com` | `janesaoirse@example.com` | Hashed: `d6670e7a92007f1b5ff785f1fc81e53aa6d3d7bd06bdf5c473cdc7286c284b6d`<br/>Base64-Encoded: `1mcOepIAfxtf94Xx/IHlOqbT170GvfXEc83HKGwoS20=` |
+| `MyEmail@example.com`<br/>`MYEMAIL@example.com` | `myemail@example.com` | Hashed: `16c18d336f0b250f0e2d907452ceb9658a74ecdae8bc94864c23122a72cc27a5`<br/>Base64-Encoded: `FsGNM28LJQ8OLZB0Us65ZYp07NrovJSGTCMSKnLMJ6U=` |
+| `My.Email@example.com` | `my.email@example.com` | Hashed: `e22b53bc6f871274f3a62ab37a3caed7214fc14d676215a96a242fcfada1c81f`<br/>Base64-Encoded: `4itTvG+HEnTzpiqzejyu1yFPwU1nYhWpaiQvz62hyB8=` |
+| `JANESAOIRSE@example.com`<br/>`JaneSaoirse@example.com` | `janesaoirse@example.com` | Hashed: `d6670e7a92007f1b5ff785f1fc81e53aa6d3d7bd06bdf5c473cdc7286c284b6d`<br/>Base64-Encoded: `1mcOepIAfxtf94Xx/IHlOqbT170GvfXEc83HKGwoS20=` |
+| `Jane.Saoirse@example.com` | `jane.saoirse@example.com` | Hashed: `	b196432c7b989a2ca91c83799957c515da53e6c13abf20b78fea94f117e90bf8`<br/>Base64-Encoded: `sZZDLHuYmiypHIN5mVfFFdpT5sE6vyC3j+qU8RfpC/g=` |
```
Comment (jon8787) [inline docs/getting-started/gs-normalization-encoding.md:120]: maybe we put all lower case `jane.saoirse@example.com` in the left column too

**Review by jon8787 — APPROVED** [review]

File: docs/getting-started/gs-normalization-encoding.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/getting-started/gs-normalization-encoding.md]: Done, thx.

File: docs/getting-started/gs-normalization-encoding.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/getting-started/gs-normalization-encoding.md:121]: OK

**Review by sunnywu — APPROVED** [review]

**Pull request merged**